Coding Guidelines.md

```diff
@@ -0,0 +1,16 @@
+#### C++
```

Comment:
Small nit on markdown style.  This is a fourth-level header. Use a single \# to start a section, then multiple \#\# within to make sub-sections. So this would be:

```md
# C++

Content goes here

# Markdown

Content goes here

# Other Languages

Content goes here
```

Reply:
Might start with a "General" section, like:

```md
# General

Project Reunion prefers using industry-standard coding styles, guidelines, and
patterns for any languages used in implementation or testing.
```

Reply:
done.